test/test_dynamic_transform.cpp

```diff
@@ -63,6 +63,10 @@ TEST_F(NVFuserTest, DynamicTransform1_CUDA) {
     expr_eval.bind(tv0->axis(1)->extent(), 3L);
     expr_eval.bind(reshape_shape0, 3L);
     expr_eval.bind(reshape_shape1, 4L);
+    // We cannot infer the shape of tv1 from the above bound values, since
+    // either axis of tv2 might be broadcast against one from tv1.
+    expr_eval.bind(tv1->axis(0)->extent(), 3L);
+    expr_eval.bind(tv1->axis(1)->extent(), 4L);
```

Comment:
What error would there be if the additional bindings were not done?

Reply:
Since the IterDomains are Symbolic, they do not exact map, so we will get an error that `expr_eval` cannot evaluate the size of `tv1` if we don't add this here. We need to evaluate that size for concretization in order to check whether it is empty.

Reply:
Hmm, but `reshape_shape0` and `reshape_shape1` are also bound. Aren't we using them as the extents of `tv1`?

Reply:
Not of `tv1`. The reshape is `tv2` and those are its extents, but `tv1` is another input that will wind up exact mapped to `tv2`.

Reply:
That makes sense. Thanks!